razzle.config.js

```diff
@@ -8,15 +11,21 @@ module.exports = {
         A prod build will fail if the API changes so it is fairly safe.
       */
       appConfig.module.rules.shift();
-      // This is to pass the bundlde performance test in CI
-      appConfig.performance = Object.assign(
-        {},
-        {
-          maxAssetSize: 350000,
-          maxEntrypointSize: 350000,
-        },
+    } else if (target === 'node' && dev) {
+      appConfig.plugins.push(
+        new BundleAnalyzerPlugin({
+          openAnalyzer: false,
```

Comment:
In the PR description, you mention this will be the static version, not the server version. However, these settings open the server at the 8888 port.  To get the static output file generated you'll need to pass in these values as well: 
```
analyzerMode: 'static',
generateStatsFile: true,
```